data/40-gdm.rules

```diff
@@ -0,0 +1,8 @@
+polkit.addRule(function(action, subject) {
```

Comment:
Yes, but it wouldn't make sense to ship this file with gdm unless the logic for password resets also moves to gdm.